.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2022-02-02
```

Comment:
Please, update the [toolchain file too](https://github.com/teloxide/teloxide/blob/master/rust-toolchain.toml)

Reply:
Done.